src/SimulationMap.cuh

```diff
@@ -12,25 +12,25 @@ class SimulationMap
 {
 public:
     /**
-     * Creates a `SimulationMap` object
+     * Creates a `SimulationMap` object and a grid of nodes
      *
-     * This function isn't implemented yet, neither it's ready to be implemented, so the description stays
-     * empty for now
+     * @param polyhedron The polyhedron in simulation
      */
-    __device__ SimulationMap(...);
+    __device__ SimulationMap(Polyhedron *polyhedron);
```

Comment:
Codacy found an issue: [Class 'SimulationMap' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/physarumAdv/minds_crawl/pullRequest?prid=5789035)